fitbenchmarking/core/tests/test_fitting_benchmarking.py

```diff
@@ -276,8 +277,9 @@ def test_perform_fit_runtime_warning(self, mock1, mock2, mock3):
                       checkpointer=self.cp)
 
             _ = fit._Fit__perform_fit(controller)
-            assert ("ratio of the max time to the min is 5.0"
-                    in ''.join([str(w[i].message) for i in range(0, len(w))]))
+            self.assertTrue("ratio of the max time to the min is 5.0"
+                            in ''.join([log.output[i]
+                                        for i in range(0, len(log.output))]))
```

Comment:
```suggestion
            self.assertTrue(("The ratio of the max time to the min is "
                             "5.00000000, which is larger than the tolerance "
                             "of 4. The min time is 1.00000000.")
                            in log.output[0])
```
You can check the whole statement to verify that the ratio and min time are printing to the right number of significant figures. I also removed the join statement because the information is in the zeroth index. The redundant join step can be removed without effecting the test.